File: COM_SIC_Activaciones/SmartWatch_Rest/BElistaOpcional.vb

Imports System.Runtime.Serialization
<Serializable()> _
Public Class BElistaOpcional

    Private _campo As String
    Private _valor As String

    Public Property campo() As String
        Get
            Return _campo
        End Get
        Set(ByVal Value As String)
            _campo = Value
        End Set
    End Property


    Public Property valor() As String
        Get
            Return _valor
        End Get
        Set(ByVal Value As String)
            _valor = Value
        End Set
    End Property
End Class

File: COM_SIC_Entidades/DataPowerRest/HeadersGenerics/BEListaOpcional.cs
using System;

namespace COM_SIC_Entidades.DataPowerRest.HeadersGenerics
{
	[Serializable]
	public class BEListaOpcional
	{

		private string _campo;
		private string _valor;


		public string campo { get { return _campo; } set { _campo = value; } }

		public string valor { get { return _valor; } set { _valor = value; } }

		public BEListaOpcional()
		{
			_campo = string.Empty;
			_valor = string.Empty;
		}
		
	}
}
